hydra-module.nix

```diff
@@ -62,15 +130,16 @@ in
       };
 
       package = mkOption {
-        type = types.path;
-        #default = pkgs.hydra;
+        type = types.package;
```

Comment:
You should probably avoid making unrelated breaking changes when submitting a large PR, because if this change breaks anything we can't easily roll back